elasticsearch/install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash 
+
+set -ex
+
+rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch
+yum install -y --setopt=tsflags=nodocs \
+  java-1.8.0-openjdk \
+  elasticsearch
+yum clean all
+
+# need these so elasticsearch doesn't reject its config while installing plugins
+export SERVICE_DNS=placeholder
+export CLUSTER_NAME=placeholder
+
+mkdir -p ${HOME}
+ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch
+/usr/share/elasticsearch/bin/plugin -i com.floragunn/search-guard/0.5
+#/usr/share/elasticsearch/bin/plugin -i openshift-elasticsearch-plugin -u http://file.rdu.redhat.com/jcantril/openshift-elasticsearch-plugin-0.4-SNAPSHOT.zip
+/usr/share/elasticsearch/bin/plugin -i openshift-elasticsearch-plugin -u http://file.rdu.redhat.com/lmeyer/openshift-elasticsearch-plugin-0.4-SNAPSHOT.zip
```

Comment:
yeah, i guess to merge we can anticipate the proper release.

On Wed, Oct 14, 2015 at 11:09 AM, Eric Wolinetz <notifications@github.com>
wrote:

> In elasticsearch/install.sh
> <https://github.com/openshift/origin-aggregated-logging/pull/8#discussion_r42006192>
> :
>
> > +
> > +rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch
> > +yum install -y --setopt=tsflags=nodocs \
> > +  java-1.8.0-openjdk \
> > +  elasticsearch
> > +yum clean all
> > +
> > +# need these so elasticsearch doesn't reject its config while installing plugins
> > +export SERVICE_DNS=placeholder
> > +export CLUSTER_NAME=placeholder
> > +
> > +mkdir -p ${HOME}
> > +ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch
> > +/usr/share/elasticsearch/bin/plugin -i com.floragunn/search-guard/0.5
> > +#/usr/share/elasticsearch/bin/plugin -i openshift-elasticsearch-plugin -u http://file.rdu.redhat.com/jcantril/openshift-elasticsearch-plugin-0.4-SNAPSHOT.zip
> > +/usr/share/elasticsearch/bin/plugin -i openshift-elasticsearch-plugin -u http://file.rdu.redhat.com/lmeyer/openshift-elasticsearch-plugin-0.4-SNAPSHOT.zip
>
> should probably change this to be the proper location, we can update the
> version after we publish our latest
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/8/files#r42006192>
> .
>